packages/shared/src/components/Feed.tsx

```diff
@@ -198,15 +203,17 @@ export default function Feed<T>({
 
   const showScrollOnboardingVersion =
     sidebarRendered &&
+    feedName === 'popular' &&
```

Comment:
I know this is hardcoded but currently it's like that in the rest of the codebase. Could be beneficial to add some kind of enum for feed types we have?

Reply:
I think we have an we introduced one recently, it should be named `enum FeedPage`.

Reply:
Correct. `MainFeedPage.Popular`